target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -358,6 +358,11 @@ youku-yk1)
 	set_wifi_led "$board:blue:air"
 	set_usb_led "$board:blue:usb"
 	;;
+kng_rc)
+	ucidef_set_led_default "power" "power" "$board:green:power" "1"
```

Comment:
Shouldn't be required. The power led is set to permanent on in diag.sh => set_state => done